### DISCUSSION TITLE
✅ [IMPLEMENTED] Make log labels list searchable

### DISCUSSION THREAD

**Discussion by molsza** [grafana/grafana#51498] [Observability]
When user has multiple log labels and he is selecting labels and values from the lists like below, then he needs scrolls a lot. 

It would be nice to be able start typing and lists of labels and then its values would shrink to only matching values. 

![image](https://user-images.githubusercontent.com/41799777/176187139-7fde0267-fce4-484e-94ac-e749694ecc66.png)

**Comment by ivanahuckova** [comment]
Hello @molsza, you are using old version of Grafana. In new version, v9.0+, you can find searchable log browser and also log label list.
<img width="737" alt="image" src="https://user-images.githubusercontent.com/30407135/178317315-a8b58ed7-7df7-4a83-b5f0-fd477f46651a.png">
